$NetBSD: patch-am,v 1.1 2000/07/26 15:33:24 wennmach Exp $

NetBSD/pmax uses "mipsel" as host cpu description.
So make make-process.o.sh grok "mipsel".

--- lwp/make-process.o.sh.in.orig	Wed Jul 26 14:58:46 2000
+++ lwp/make-process.o.sh.in		Wed Jul 26 14:59:06 2000
@@ -48,7 +48,7 @@
     sparc*) PROCESS_S="process.sparc.S" ;;
     i*86*)  PROCESS_S="process.i386.S" ;;
     m68k)   PROCESS_S="process.m68k.S" ;;
-    mips)   PROCESS_S="process.mips.S" ;;
+    mips*)   PROCESS_S="process.mips.S" ;;
     alpha*)  PROCESS_S="process.alpha.S" ;;
     hppa*)  PROCESS_S="process.hpux.S" ;;
     powerpc)  PROCESS_S="process.ppc.S" ;;
